integration-tests/suites/runtime_config_file.go

```diff
@@ -0,0 +1,195 @@
+package suites
+
+import (
+	"fmt"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	"github.com/stackrox/collector/integration-tests/pkg/assert"
+	"github.com/stackrox/collector/integration-tests/pkg/collector"
+	"github.com/stackrox/collector/integration-tests/pkg/common"
+	"github.com/stackrox/collector/integration-tests/pkg/config"
+	"github.com/stackrox/collector/integration-tests/pkg/types"
+)
+
+var (
+	normalizedIp = "255.255.255.255"
+	externalIp   = "8.8.8.8"
+	serverPort   = 53
+	externalUrl  = fmt.Sprintf("http://%s:%d", externalIp, serverPort)
+
+	activeNormalizedConnection = types.NetworkInfo{
+		LocalAddress:   "",
+		RemoteAddress:  fmt.Sprintf("%s:%d", normalizedIp, serverPort),
+		Role:           "ROLE_CLIENT",
+		SocketFamily:   "SOCKET_FAMILY_UNKNOWN",
+		CloseTimestamp: types.NilTimestamp,
+	}
+
+	activeUnnormalizedConnection = types.NetworkInfo{
+		LocalAddress:   "",
+		RemoteAddress:  fmt.Sprintf("%s:%d", externalIp, serverPort),
+		Role:           "ROLE_CLIENT",
+		SocketFamily:   "SOCKET_FAMILY_UNKNOWN",
+		CloseTimestamp: types.NilTimestamp,
+	}
+
+	inactiveNormalizedConnection = types.NetworkInfo{
+		LocalAddress:   "",
+		RemoteAddress:  fmt.Sprintf("%s:%d", normalizedIp, serverPort),
+		Role:           "ROLE_CLIENT",
+		SocketFamily:   "SOCKET_FAMILY_UNKNOWN",
+		CloseTimestamp: "Not nill time",
+	}
+
+	inactiveUnnormalizedConnection = types.NetworkInfo{
+		LocalAddress:   "",
+		RemoteAddress:  fmt.Sprintf("%s:%d", externalIp, serverPort),
+		Role:           "ROLE_CLIENT",
+		SocketFamily:   "SOCKET_FAMILY_UNKNOWN",
+		CloseTimestamp: "Not nill time",
+	}
+
+	runtimeConfigDir  = "/tmp/collector-test"
+	runtimeConfigFile = filepath.Join(runtimeConfigDir, "/runtime_config.yaml")
+	collectorIP       = "localhost"
+)
+
+type RuntimeConfigFileTestSuite struct {
+	IntegrationTestSuiteBase
+	ClientContainer string
+}
+
+func (s *RuntimeConfigFileTestSuite) createDirectory(dir string) {
+	cmd := "mkdir " + dir
+	s.execShellCommand(cmd)
+}
+
+func (s *RuntimeConfigFileTestSuite) deleteFile(file string) {
+	cmd := "rm " + file
+	s.execShellCommand(cmd)
+}
+
+func (s *RuntimeConfigFileTestSuite) setRuntimeConfig(runtimeConfigFile string, configStr string) {
+	cmd := "echo '" + configStr + "' > " + runtimeConfigFile
+	s.execShellCommand(cmd)
+}
+
+func (s *RuntimeConfigFileTestSuite) setExternalIpsEnable(runtimeConfigFile string, enable bool) {
+	configStr := `networking:
+  externalIps:
+    enable: ` + strconv.FormatBool(enable)
+	s.setRuntimeConfig(runtimeConfigFile, configStr)
```

Comment:
It would be better for to construct the object and serialize it to a string, and in fact it may be best to have the caller pass the structure directly to setRuntimeConfig

```
func (s *RuntimeConfigFileTestSuite) setRuntimeConfig(path string, config types.RuntimeConfig) error
```

Reply:
Done